Fixes broken `foxsi_get_output_image_cube`

ayshih opened the pull request:
Fixes #22 

`foxsi_get_output_image_cube` now calls `foxsi_get_output_2d_image` (as it should) to avoid unnecessary code duplication.

ayshih committed:
Refactored foxsi_get_output_image_cube to call foxsi_get_output_2d_image

ayshih committed:
Fixed name of variable

aringlis commented:
When I test this on the default input cube, for some reason the first and last output maps (0 and 29) have data all zero.

ayshih commented:
Hmm, that's because the effective areas for those two energy bins are 5e-11 and 0, respectively.  If you turn off the addition of Poisson noise (i.e., `/no_count_stats`), you can see that the first map has some ridiculously small numbers of expected counts.

LinErinG commented:
Yes, there's effectively no area in the lowest bin of the default cube (it's ~1-3 keV).  There's a tiny bit of EA for the uppermost bin (58-60) but it's effectively zero.

LinErinG commented:
I tested out the pull request and it looks good to me.

ayshih committed:
Fixed bug where LUN was not properly freed

ayshih committed:
Changed the default image cube to have nicer energy binning

ayshih commented:
I've changed the default image cube to have energy binning from 2 to 60 keV in exact steps of 2 keV.  I also fixed a bug where the logical units for opening files weren't being freed.

aringlis commented:
Looks good :+1:
